harvester/dashboard
# Harvester: Add restart dialog for vm restarting

### Summary
Add restart dialog


### Occurred changes and/or fixed issues


- https://github.com/harvester/harvester/issues/2498

### Technical notes summary


### Areas or cases that should be tested




#### Test steps
1. Create a vm
2. Go to edit vm
3. Click Save
4. VM restart dialog should be visible
5. Click Save & Restart
6. VM should be saved and restarted
7. Click Save
8. VM should be only saved

### Areas which could experience regressions


### Screenshot/Video
